install-and-launch.sh

```diff
@@ -478,15 +482,49 @@ install_sublime() {
         exit 1
     fi
 
-    print_success "** Successfully installed Sublime Platform! **"
+}
+
+health_check() {
+    pg_error_string="retryable migration error: pq: password authentication failed for user"
+    bora_container_id="$(container_id_by_name "sublime_bora_lite")"
+
+    abort=false
+
+    echo
+    echo "Checking health of containers..."
+    sleep 5
+
+    if [ -z "$bora_container_id" ]; then
+        echo "error: bora container not found. Stopping containers..."
+        exit 1
+    fi
 
-    dashboard_url=$(grep 'DASHBOARD_PUBLIC_BASE_URL' sublime.env | cut -d'=' -f2)
-    printf "\nIt may take a couple of minutes for all services to start for the first time.\n\n"
-    printf "For info on how to start or stop the Platform, or help with troubleshooting, see the docs:\n\n"
-    printf "https://docs.sublimesecurity.com/docs/quickstart-docker#troubleshooting\n"
-    print_success "Your Sublime Dashboard: $dashboard_url"
+    bora_logs="$(docker logs "$bora_container_id")"
```

Comment:
could move the grep here, cause if there are a lot of logs, you don't need to pull them all into a variable